components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -296,7 +296,7 @@ be lost if SCREAM_HACK_XML is not enabled.
 
       <spa_data_file type="file" doc="File containing aerosol data. Must be on same grid as the atm, or a coarser one"/>
       <spa_data_file hgrid="ne.*np4">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc</spa_data_file>
-      <spa_data_file hgrid="ne.*np4.pg2">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30pg2_20240111.nc</spa_data_file>
+      <spa_data_file hgrid="ne.*np4.pg2">${DIN_LOC_ROOT}/atm/scream/init/spa_v3.LR.F2010.2011-2025.c_20240405.nc</spa_data_file>
```

Comment:
Should we maybe add the grid name to the file? It may help to recognize immediately what the resolution is, without having to open the file...

Reply:
We can't remap ne30->ne4 _online_. Unfortunately, the online remappers hard coded the assumption that the fine grid is the model grid (so you can coarsen output, or read in coarse inputs). It is of course possible to extend them, but when I tried I quickly realized it wasn't going to be super quick, so I gave up (since we didn't really need it).

For this case (ne4 input) I would simply remap offline, and upload the new ne4 spa data on the server.

Reply:
I'm always happy if someone else does the work... ;-)